.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: "@oat-sa/eslint-config-tao",
```

Comment:
suggestion:
we can use the `node` profile and maybe in the future we can add a `cypress` profile in the eslint config
 ```suggestion
    extends: "@oat-sa/eslint-config-tao/node",
```

Reply:
and `Cypress` should be in the globals no ?

Reply:
Not 100% sure about using the Node profile. As I understand it, Cypress code is running partly in a Node process, partly in a browser environment. (I set `eslint-config-tao/node`, and had a lint error that `btoa is not defined` (lti.js L73) - as if in Node.js. However it _is_ defined in the test, and produces a valid hash.)

I'm on board for making the custom profile.

Reply:
yes, let's check that separately